src/index.html

```diff
@@ -4,9 +4,35 @@
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Document</title>
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bulma@0.8.2/css/bulma.min.css">
+  <link rel="stylesheet" href="style.css">
+  <title>social-network-recipes</title>
```

Comment:
Regularmente una sola palabra puede describir mejor el titulo
```suggestion
  <title>recipes</title>
```